x-pack/plugins/session_view/public/components/session_view/styles.ts

```diff
@@ -35,11 +37,22 @@ export const useStyles = ({ height = 500 }: StylesDeps) => {
       margin: `${euiTheme.size.m} ${euiTheme.size.xs} !important`,
     };
 
+    const sessionViewerComponent: CSSObject = {
+      border: euiTheme.border.thin,
+      borderRadius: euiTheme.border.radius.medium,
+    };
+
+    const toolBar: CSSObject = {
+      backgroundColor: `#EEF2F7 !important`,
```

Comment:
same consideration about if the `!important` is really needed here

Reply:
I updated the HEX color code to use EUI variable instead, but I'm keeping the !important, as the design requires me to use colors outside the colors available from the EuiPanel color props